lib/abao.coffee

```diff
@@ -40,7 +40,7 @@ class Abao
       # Parse tests from RAML
       (raml, callback) ->
         if !config.options.server
-          if 'baseUri' in raml
```

Comment:
Why was this changed? It was coded as such to denote that `baseUri` is an optional field, per [RAML specs](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#base-uri-and-base-uri-parameters)